ansible/roles/gen_kubernetes_config/templates/beacon-metrics-gazer.yaml.j2

```diff
@@ -0,0 +1,57 @@
+# ANSIBLE GENERATED FILE - DO NOT EDIT MANUALLY
+beacon-metrics-gazer:
+  image:
+    repository: dapplion/beacon-metrics-gazer
+    tag: v0.1.3
+
+  fullnameOverride: beacon-metrics-gazer
+  resources:
+    requests:
+      cpu: 68m
+      memory: 128Mi
+    limits:
+      cpu: 128m
+      memory: 256Mi
+
+  serviceMonitor:
+    enabled: true
+    labels:
+      vmagent: common
+      network: "{{ ethereum_network_name }}"
+
+  customCommand:
+    - /beacon-metrics-gazer
+    - http://lodestar-nethermind-001:5052
```

Comment:
Should probably make this URL configuriable in `all.yaml` or something. So that we don't need to mess with the gen and it won't fail if we ever switch the k8s node combo from lodestar-nethermind.

Reply:
Right, I'll use a template for all this, good point.